documentation/samplefiles/readme.typescript.md

```diff
@@ -6,8 +6,8 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ```yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "YourServiceName"
-  output-folder: "$(typescript-sdks-folder)/packages/YourServiceName"
+  package-name: "[[ServiceName]]"
+  output-folder: "$(typescript-sdks-folder)/packages/[[ServiceName]]"
```

Comment:
If it's management sdk, the packkage name convention should be like this and the sdk output-folder should be like this.   
  package-name: "@azure/arm-datafactory"
  output-folder: "$(typescript-sdks-folder)/sdk/datafactory/arm-datafactory"

Reply:
Also, please add clear-output-folder=true.

Reply:
@qiaozha  are the "@azure/arm-" and "arm- " fixed words? Or [[ReleaseState]] eg:"preview""stable", you can view the placeholder meaning in https://github.com/JackTn/azure-rest-api-specs/blob/dev-tianenattestation-Microsoft.TianenAttestation-2020-09-27/specification/tianenattestation/resource-manager/readme.csharp.md

Reply:
They are fixed words for mamanegement plane.